### ISSUE TITLE
ngf-ratio not working??

### ISSUE THREAD

**Issue by armangevorgyan** [danialfarid/ng-file-upload#1169]
ngf-ratio="8" not working ?? maybe because in JS 8000/1000 = 7.994082840236686, 
and how i can use it for 0.3 < width/height < 8 ??
and round ratio? ( 7.99 -> 8 or 0.35 ->0.3(0.4)) 
can i ask why height=1000px width=8000px => console show ngfWidth = 1351, ngfHeight = 169,
Thank you. Sorry for bad English ))

**Comment by danialfarid** [comment]
Post your code here, the plugin should take care of rounding.

**Comment by armangevorgyan** [comment]
<md-button class="button" accept="image/*"
                           ngf-select="main.uploadImages($files)"
                           name="browse"
                           ngf-pattern="'.jpg,.png,.jpeg'"
                           ngf-max-size="50MB"
                           ngf-max-height="10000"
                           ngf-min-height="100"
                           ngf-min-width="100"
                           ngf-max-width="10000"
                           ng-model="files"
                           multiple="multiple"
                           ngf-ratio="0.3,0.4,0.5,0.6,0.7,0.8,0.9,1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9"
  ng-model-options="{allowInvalid: true}"  >
                    Browse
                </md-button>

**Comment by armangevorgyan** [comment]
console.log('Valid images', 'ratio', file.$ngfWidth / file.$ngfHeight, 'width', file.$ngfWidth, "height", file.$ngfHeight)
console: Valid images ratio 7.994082840236686 width 1351 height 169

**Comment by danialfarid** [comment]
why don't you use `ngf-min-ratio` and `ngf-max-ratio`?

**Comment by armangevorgyan** [comment]
I try it but didn't catch error, and about Width,  height??
try this 
ngf-min-ratio="3:10"
 ngf-max-ratio="8:1"
and this
 ngf-min-ratio="0.3"
 ngf-max-ratio="8"

**Comment by danialfarid** [comment]
Can you create a jsfiddle?

**Comment by danialfarid** [comment]
Also it is 
```
ngf-min-ratio="'3:10'"
ngf-max-ratio="'8:1'"
and this
ngf-min-ratio="'0.3'"
ngf-max-ratio="'8'"
```

**Comment by danialfarid** [comment]
Alright seems like there is a bug with min max ratio. 
I'll fix it in the next release.

**Comment by armangevorgyan** [comment]
Sorry but all mixed cant crate jsfiddle. 
What you can saw about width Height??

**Comment by danialfarid** [comment]
ok `ngf-min-ratio' and `ngf-max-ratio` are fixed at version 10.1.1, you can use those now.

**Comment by abhimanyusingh** [comment]
max and min ratio is not working here is the fiddle, I am trying to upload an image size: 1200*627, but I am getting following error
`{"$ngfBlobUrl":"blob:http://fiddle.jshell.net/a762e38a-3543-4661-ae2a-ddb1257f8317","$ngfWidth":1200,"$ngfHeight":627,"$error":"ratio","$errorMessages":{"ratio":true},"$errorParam":"1.90:1"} `
http://jsfiddle.net/maqbzv15/5353/


can some let me know what I am doing wrong here
